Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2_test.py

```diff
@@ -1520,3 +1522,31 @@ def test_update_remote_system_command(mocker, prisma_cloud_v2_mirroring_client,
 
     assert mock_update_remote_alert.call_count == expected_call_count
     assert result == 'P-1111111'
+
+
+def test_remove_additional_resource_fields(prisma_cloud_v2_client):
+    """
+        Given
+            - Results of config_search_command.
+        When
+            - Running the config_search_command.
+        Then
+            - Verify that remove_additional_resource_fields removes only the required fields.
+    """
+    from PrismaCloudV2 import remove_additional_resource_fields
+    input = {
+        'a': 'a_val',
+        'b': {
+            'c': 'c_val',
+            'd': 'd_val',
+            'e': 'e_val'
+        }
+    }
+    expected = {
+        'a': 'a_val',
+        'b': {
+            'e': 'e_val'
+        }
+    }
```

Comment:
Lets add a case for what we are actually removing
```suggestion
    input = {
        'a': 'a_val',
        'b': {
            'c': 'c_val',
            'd': 'd_val',
            'e': 'e_val'
        },
        'items':{'data':{'metadata':{'items':{'configure-sh':'c_val','not-removed':'n_val'}},'disks':{'shieldedInstanceInitialState':'s_val'}}}
    }
    expected = {
        'a': 'a_val',
        'b': {
            'e': 'e_val'
        },
        'items':{'data':{'metadata':{'items':{'not-removed':'n_val'}}}}
    }
```